docs/Usage/CellMarkers.md

```diff
@@ -0,0 +1,73 @@
+
+### Cell Markers
+
+A cell marker is a special inline comment that defines the lines that begin each cell. The inline comment must start with the comment character registered in the language's grammar file.
+
+- Python: `#`
+- R: `#`
+- Julia: `#`
+- Javascript: `//`
+- C++: `//`
+- Stata: `//`
+- Matlab: `%`
+
+#### Code Cells
+
+After the comment character, you need to add one space plus any of the following markers on the same line:
```

Comment:
Could use better explanation of how cell headers are formed. **Note:** a same is not required between the comment characters and the start of the cell header.

Reply:
> Could use better explanation of how cell headers are formed. 

I tried to rephrase it but it can still be improved let me know if you have something

>**Note:** a same is not required between the comment characters and the start of the cell header.

I'm assuming this meant that the said 'space' is not required